tools/cbmc/proofs/DNS/DNSgetHostByName_a/cbmc-batch.yaml

```diff
@@ -0,0 +1,4 @@
+jobos: ubuntu16
+cbmcflags: '=--object-bits;7;--32;--bounds-check;--pointer-check;--unwind;1;--unwindset;prvProcessDNSCache.0:5,strlen.0:11,__builtin___strcpy_chk.0:11,strcmp.0:11,xTaskResumeAll.0:11,xTaskResumeAll.1:11;--nondet-static;--unwinding-assertions='
```

Comment:
@markrtuttle here's another reason not to commit these files. The `Makefile.json` has a nice variable called `HOSTNAME_UNWIND` which is computed as `1 + MAX_HOSTNAME_LEN`, and the variable is used throughout the file. But in `cbmc-batch.yaml`, we just have the magic number `11` everywhere. This will certainly go out of date if we update `Makefile.json`, I highly doubt we'll remember to change `11` to the correct value.